File: eureka-client-admin/src/main/java/com/javaweb/web/eo/roleRestrict/RoleRestrictListRequest.java
package com.javaweb.web.eo.roleRestrict;

import java.io.Serializable;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class RoleRestrictListRequest implements Serializable {

	private static final long serialVersionUID = 330607673594076826L;
	
	private String roleName;//角色名称
	
	private String province;//省（不限为all，本省为self）
	
	private String city;//市（不限为all，本市为self）
	
	private String district;//区（不限为all，本区为self）
	
	private Long currentPage = 1L;//默认当前第1页
	
	private Long pageSize = 10L;//默认每页显示10条

}

File: angular-admin/src/app/model/roleRestrict/RoleRestrictListRequest.ts
import {CommonConstant} from '../../constant/CommonConstant';

export class RoleRestrictListRequest {

  roleName:string = CommonConstant.EMPTY;//角色名

  province:string = CommonConstant.EMPTY;//省（不限为all，本省为self）

  city:string = CommonConstant.EMPTY;//市（不限为all，本市为self）
  
  district:string = CommonConstant.EMPTY;//区（不限为all，本区为self）

  currentPage = 1;//当前页数

  pageSize = 10;//每页显示多少条

}
